radiacode/radiacode.py

```diff
@@ -25,7 +25,11 @@ def __init__(
         ignore_firmware_compatibility_check: bool = False,
     ):
         self._seq = 0
-        if bluetooth_mac is not None:
+
+        # Bluepy doesn't support MacOS: https://github.com/IanHarvey/bluepy/issues/44
+        self._bt_supported = False if platform.system() == 'Darwin' else True
```

Comment:
`self._bt_supported = platform.system() != 'Darwin'`

Reply:
Change applied and pushed in the branch